src/App.tsx

```diff
@@ -18,11 +19,11 @@ function App() {
         <Sidebar>
           <Routes>
             <Route path="/" element={<Outlet />}>
-              <Route path="/dashboard" element={<h1>dashboard</h1>}></Route>
-              <Route path="/transfer" element={<h1>transfer</h1>}></Route>
-              <Route path="/history" element={<h1>history</h1>}></Route>
-              <Route path="/bills" element={<h1>bills</h1>}></Route>
-              <Route path="*" element={<h1>Not Found</h1>}></Route>
+              <Route path="dashboard" element={<Dashboard />}></Route>
```

Comment:
if you have conflicting paths, just pass the `exact` prop for the route that contains the common part of the path. In this case line 21.

Reply:
exact has been removed from react router 6